templates/files/k8s-resource/kube-proxy-ds.yaml

```diff
@@ -25,9 +25,7 @@ spec:
         scheduler.alpha.kubernetes.io/critical-pod: ''
     spec:
       tolerations:
-      - key: node-role.kubernetes.io/master
-        operator: Exists
-        effect: NoSchedule
+        - operator: Exists
```

Comment:
not sure why this was an issue, but somehow kube-proxy did was not scheduled on not-ready nodes and it caused  deadlock since  aws-cni needed access to k8s api in order install CNI

Reply:
so adding global ignore all taints